npc/custom/official/lasagna/warps.txt

```diff
@@ -0,0 +1,121 @@
+lasa_fild02,348,244,0	warp	lasa_fild02#lasa_dun01	1,1,lasa_dun01,24,143
+lasa_dun01,18,143,0	warp	lasa_dun01#lasa_fild02	1,1,lasa_fild02,344,243
+
+lasa_dun_q,190,14,0	warp	lasa_dun_q#lasa_dun01	1,1,lasa_dun01,152,98
+lasa_dun01,157,98,0	warp	lasa_dun01#lasa_dun02	1,1,lasa_dun02,22,171
+lasa_dun02,18,171,0	warp	lasa_dun02#lasa_dun01	1,1,lasa_dun01,153,98
+
+lasa_dun02,146,58,0	warp	lasa_dun02#lasa_dun03	1,1,lasa_dun03,190,18
+lasa_dun03,190,15,0	warp	lasa_dun03#lasa_dun02	1,1,lasa_dun02,146,54
+
+lasa_fild01,134,381,0	warp	lasa_fild01#lasagna_1	2,2,lasagna,153,58
+lasagna,150,54,0	warp	lasagna#lasa_fild01_1	2,2,lasa_fild01,131,378
+
+lasa_fild01,341,375,0	warp	lasa_fild01#lasagna_2	2,2,lasagna,327,56
+lasagna,327,51,0	warp	lasagna#lasa_fild01_2	2,2,lasa_fild01,344,371
+
+lasagna,358,91,0	warp	lasagna#lasa_fild02	2,2,lasa_fild01,20,98
+lasa_fild02,16,98,0	warp	lasa_fild02#lasagna	2,2,lasagna,355,92
+
+// Boat
+conch_in,60,61,0	script	warp_conch_in#out	WARPNPC,1,1,{
+OnTouch:
+	warp "lasagna",206,323;
```

Comment:
Can we implement warp2 to work like this instead of coding this special case which is more likely to be the behavior of warps in the future.